source/developer_manual/design/code.rst

```diff
@@ -0,0 +1,242 @@
+.. _code_design:
```

Comment:
is this from confluence?

Reply:
no, this is the code design page from wiki. It is mostly redundant with the coding style guide which is also imported, i was thinking of replacing this one completely with Coding style guide. But didn't go for it yet, what do you think should i just replace it?

Reply:
I think we should use the confluence one only.